File: exs/mundo_1/c++/016.cpp
/*

016 - Quebrando um número
  Crie um programa que leia um número real qualquer pelo teclado e mostre na
  tela a sua porção inteira

*/

#include <iostream>

using namespace std;

int main() {
  double numero_real;

  cin >> numero_real;
  cout << (int) numero_real << "\n";

  return 0;
}

File: exs/mundo_1/rust/016.rs
/*
Desafio 016

Problema: Crie um programa que leia um número Real qualquer
          pelo teclado e mostre na tela a sua porção Inteira.
*/

fn main() {
    let mut valor = String::new();

    std::io::stdin().read_line(&mut valor).unwrap();

    let valor = valor
        .trim()
        .parse::<f32>()
        .expect("O valor não é um número");

    println!("{}", valor as i32);
}
